fhnw-ph-schreibberatung.csl

```diff
@@ -0,0 +1,476 @@
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <updated>2015-01-13T12:00:00+00:00</updated>
+    <summary>Empfehlungen der Schreibberatung PH FHNW (Paedagogische Hochschule, Fachhochschule Nordwestschweiz (CH))</summary>
+  </info>
+
+  <!-- "et al." anstatt "u.a." -->
+    <locale xml:lang="de">
+      <terms>
+        <term name="et-al">et al.</term>
+      </terms>
+    </locale>
+      
+  <macro name="secondary-contributors">
+    <choose>
+      <if type="chapter paper-conference" match="none">
+        <group delimiter=". ">
+          <choose>
+            <if variable="author">
+              <names variable="editor">
+                <label form="verb-short" text-case="capitalize-first" suffix=" " strip-periods="true"/>
+                <name and="text" delimiter=", "/>
+              </names>
+            </if>
+          </choose>
+          <choose>
+            <if variable="author editor" match="any">
+              <names variable="translator">
+                <label form="verb" text-case="capitalize-first" suffix=" " strip-periods="true"/>
+                <name and="text" delimiter=", "/>
+              </names>
+            </if>
+          </choose>
+        </group>
+      </if>
+    </choose>
+  </macro>
+
+  <macro name="container-contributors">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <group delimiter=". ">
+          <choose>
+            <if variable="author">
+              <names variable="editor">
+                <name form="long" name-as-sort-order="all" and="text" delimiter="; "  delimiter-precedes-last="never"/>
+                <label form="short" prefix=" (" suffix=".)" strip-periods="true"/>
+              </names>
+            </if>
+          </choose>
+          <choose>
+            <if variable="author editor" match="any">
+              <names variable="translator">
+                <name form="long" name-as-sort-order="all" and="text" delimiter="; "   delimiter-precedes-last="never"/>
+                <label form="short" prefix=" (" suffix=".)" strip-periods="true"/>
+              </names>
+            </if>
+          </choose>
+        </group>
+      </if>
+    </choose>
+  </macro>
+
+  <macro name="anon">
+    <choose>
+      <if variable="author editor translator" match="none">
+        <choose>
+          <if type="entry entry-dictionary entry-encyclopedia broadcast" match="none">
+            <text term="anonymous" text-case="capitalize-first"/>
+          </if>
+          <else-if type="broadcast">
+            <text variable="container-title" font-style="normal" suffix="."/>
+          </else-if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+
+  <macro name="editor">
+    <names variable="editor">
+      <name name-as-sort-order="all" and="text" sort-separator=", " delimiter="; " delimiter-precedes-last="never"/>
+      <label form="short" prefix=" (" suffix=".)" strip-periods="true"/>
```

Comment:
im Prinzip ja. Da der Punkt ja in der Abkürzung enthalten ist, wäre der aber auch noch zu entfernen, also suffix=")" hier (im Moment stehen da an sich zwei Punkte hintereinander--Zotero nimmt dann den einen automatisch raus, aber wir verlassen uns da nicht gerne drauf). Natürlich nur dort wo sie zuvor das "strip-periods" entfernt haben. Anderswo hat der Punkt im suffix ja durchaus eine legitime Funktion.
Wäre schön, wenn Sie das noch ändern würden, im Notfall geht's aber auch so.